addon-test-support/@ember/test-helpers/dom/-is-select-element.ts

```diff
@@ -0,0 +1,14 @@
+import { isDocument } from './-target';
+
+/**
+  @private
+  @param {Element} element the element to check
+  @returns {boolean} `true` when the element is a select element, `false` otherwise
+*/
+export default function isSelectElement(element: Element | Document): element is HTMLSelectElement {
+  return (
+    !isDocument(element) &&
+    element.tagName === 'SELECT' &&
+    (element as HTMLInputElement).type !== 'hidden'
```

Comment:
why does this fail for hidden selects?

and shouldn't this be a `HTMLSelectElement` instead?

Reply:
Good spot on the Element type, as for the hidden I used the existing fill in component for the logic to keep it consistent. If you feel it isnt needed then i am happy to remove. But i wanted to keep it as close to how the other helper worked